### ISSUE TITLE
Allow logged-in users to "star" pre-trained results to sort by popularity (3.1d)

### ISSUE THREAD

**Issue by billspat** [krishnanlab/geneplexus_app#137]
In the 'published' list of outputs ( #136 ), create an interface for logged-in users to see those published results, and mark them as favorites.     

- [ ] database structure to hold 'favorites' per user
- [ ] UI to star results  in published results UI from #136
- [ ] UI and or data structure to add counts of stars per result
- [ ] UI (filter?) to let user see only their favorites in list of published results
 
dependent on #134 and #136

**Comment by billspat** [comment]
features and issues deferred for rewrite --closing here.
